src/com/oltpbenchmark/api/Worker.java

```diff
@@ -592,7 +604,11 @@ protected final ArrayList<Pair<TransactionExecutionState, TransactionStatus>> do
                     break;
                 }
             } // WHILE
-            conn.close();
+            if(wrkld.getUseConnMngr()) {
+                if (wrkld.getUseShortLivedConn())
+                    conn.close();
+            } else
+                conn.close();
```

Comment:
The connection shouldn't be closed  for long lived connections

Reply:
The connections are not closed for long lived ones.